zebracrossing/campaigns/tests.py

```diff
@@ -20,6 +21,51 @@ def test_equals(self):
         self.assertEqual(time_slot, other)
 
 
+class APIEndpointTests(TestCase):
+    @classmethod
+    def setUp(self):
+        self.client = Client()
+        self.start_date = "2022-07-07"
+        self.end_date = "2022-07-08"
+        self.schedule_list = [
+            {
+                "campaign": "Telkom",
+                "material": {"download_url": "", "type": "REC"},
+                "scheduled": "2022-07-07:07:40",
+            }
+        ]
+        self.url = f"http://localhost:8000/api/v1/schedule/?start={self.start_date}&end={self.end_date}"
```

Comment:
Use relative URLs here: `/api/v1/schedule...` In general you should not be hardcoding the protocol or domain into any requests